test/strict.coffee

```diff
@@ -45,8 +45,7 @@ test "octal escape sequences prohibited", ->
   strict    '"\\\\\\1"'
   strictOk  '"\\0"'
   eq "\x00", "\0"
-  strictOk  '"\\08"'
-  eq "\x008", "\08"
+  strict    '"\\08"'
```

Comment:
I guess rather than "flipping" this test from `strictOk()` to `strict()`, I could just remove it since technically it's not a strict-mode-specific test (it'll always fail to compile regardless of strict mode or not) and is already covered by the tests in `test/error_messages.coffee` above?

Reply:
Sounds good to me!

Reply:
Ok removed